pyrato/parameters.py

```diff
@@ -102,3 +103,113 @@ def reverberation_time_linear_regression(
         return reverberation_times, intercepts
     else:
         return reverberation_times
+
+
+
+
+
+
+def clarity(energy_decay_curve, early_time_limit=80):
+    r"""
+    Calculate the clarity from the energy decay curve (EDC) of
+    a room impulse response.
+
+    The clarity parameter (C50 or C80) is defined as the ratio of early-to-late
+    arriving energy in an impulse response and is a measure for how clearly
+    speech or music can be perceived in a room. The early-to-late boundary is
+    typically set at 50 ms (C50) or 80 ms (C80) [#iso]_.
+
+    Clarity is calculated as:
+
+    .. math::
+
+        C_{t_e} = 10 \log_{10} \frac{
+            \displaystyle \int_0^{t_e} p^2(t) \, dt
+        }{
+            \displaystyle \int_{t_e}^{\infty} p^2(t) \, dt
+        }
+
+    where :math:`t_e` is the early time limit and :math:`p(t)` is the pressure
+    of a room impulse response. The clarity can also be efficiently computed
+    from the EDC directly by:
```

Comment:
```suggestion
    where :math:`t_e` is the early time limit and :math:`p(t)` is the pressure
    of a room impulse response. Here, the clarity is efficiently computed
    from the EDC as:
```
For users the information what is actually happening is more important than how it could theoretically be done ;)